LICENSE

```diff
@@ -1,5 +1,6 @@
 BSD 3-Clause License
 
+Copyright (c) 2022, Jupyter Development Team
 Copyright (c) 2022, David Brochart
```

Comment:
Should we remove the "David Brochart" copyright?

Reply:
I think it is fair to keep it as you started this repo.